### PR TITLE
[Hono] Update Bitnami Kafka chart to version 20.x

### PR DESCRIPTION
**Pull request by sophokles73**
Also configured Kafka and Zookeeper JVMs to use at most 512M memory.

### CODE REVIEW

**Comment by sophokles73** [comment]
closed in favor of #438

File: charts/hono/values.yaml [hunk 1]
```diff
@@ -1718,15 +1718,31 @@ kafka:
...
+        enabled: true
+        clientUser: "zookeeperUser"
+        clientPassword: "zookeeperPassword"
+        serverUsers: "zookeeperUser"
+        serverPasswords: "zookeeperPassword"
+    # limit Zookeeper JVM's memory consumption
+    heapSize: 400
+    resources:
+      requests:
+        memory: "500Mi"
```
Comment (calohmn) [inline charts/hono/values.yaml:1743]: How about keeping the zookeeper default of `256Mi` here for the `requests` value? I would also consider reducing the corresponding kafka resources/requests/memory configuration. Both would reduce the overall minimal memory requirements which could for example be relevant when installing the cloud2edge chart locally for testing purposes.

Comment (sophokles73) [reply charts/hono/values.yaml:1743]: I had replaced this PR with #438 ...

Comment (calohmn) [reply charts/hono/values.yaml:1743]: Sorry, mixed up the 2 PRs.

**Pull request closed without merging**